basie/valid_angles.py

```diff
@@ -43,9 +43,15 @@
 
 class VAngle(Angle):
     def __new__(cls, angle, unit=u.deg, wrap_angle=360 * u.deg, **kwargs):
+        was_tuple=False
+        if isinstance(angle, tuple) and unit in (u.deg, u.hour):
```

Comment:
The tuple of values does not work anymore with astropy's Angle in recent versions, due to its ambiguous behavior when the degree value is 0. So, I reformatted the angle as a single number in degrees when it is given as a tuple, but kept the representation as sexagesimal so that the output does not change.